code/modules/countdown/countdown.dm

```diff
@@ -108,3 +108,16 @@
 	else if(D.gang && D.gang.dom_timer)
 		var/timer = D.gang.dom_timer
 		return timer
+
+/obj/effect/countdown/clockworkgate
+	name = "gateway countdown"
+	text_size = 1
+	text_color = "#BE8700"
+	layer = POINT_LAYER
+
+/obj/effect/countdown/clockworkgate/get_value()
+	var/obj/structure/clockwork/massive/celestial_gateway/G = attached_to
+	if(!istype(G))
+		return
+	else if(G.health && !G.purpose_fulfilled)
+		return "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'>[GATEWAY_RATVAR_ARRIVAL - G.progress_in_seconds]</div>"
```

Comment:
why isn't centering the default i dunno it seems like centering by default would be useful